### PR TITLE
Version 1.13.1 release

### PR DESCRIPTION
**Pull request by sobolevn**
Let's hope that it will work!

### CODE REVIEW

**Comment by intgr** [comment]
Wow looks like there are other issues with mypy 0.990 https://github.com/typeddjango/django-stubs/actions/runs/3421536321/jobs/5697823439#step:6:462

For the pytest tests you can add `show_error_codes = False` to mypy configuration, that should fix many of the diffs.

File: requirements.txt [hunk 1]
```diff
@@ -9,4 +9,4 @@ psycopg2-binary
 -e .[compatible-mypy]
 
 # Overrides:
-mypy==0.982
+mypy==0.990
```
Comment (intgr) [inline requirements.txt]: Unless mypy 0.991 solves the issues we're having, IMO we can make a release that's being tested with `mypy==0.982`.

Having a release that solves *some* mypy 0.99x problems is better than delaying the release.

Of course having full support for latest mypy would be ideal, but right now I don't have time and interest to look into the CI failures.

**Comment by intgr** [comment]
I opened bug #1261 about the plugin internal error with mypy 0.99x.

Let's defer mypy update for now and make a release anyway?

**Comment by sobolevn** [comment]
@intgr would you like to do the release? :)

**Comment by intgr** [comment]
Sure, I can give it a shot. What are the exact commands I should run; is there documentation somewhere?

**Comment by sobolevn** [comment]
https://github.com/typeddjango/django-stubs/blob/master/scripts/release.sh

**Comment by sobolevn** [comment]
Plus `git tag`

**Comment by intgr** [comment]
Looks like I may have done it 😮 

* https://github.com/typeddjango/django-stubs/releases/tag/1.13.1
* https://pypi.org/project/django-stubs/#history

I will clean up the autogenerated release notes soon.

**Comment by sobolevn** [comment]
Congrats 🎉

**Pull request merged**